scripts/liminal

```diff
@@ -124,10 +125,17 @@ def start():
     if docker_is_running():
         # initialize liminal home by default
         environment.get_liminal_home()
-        result = subprocess.call(
-            [f'docker-compose -f "{environment.get_liminal_home()}/docker-compose.yml" up -d'],
-            env=os.environ, shell=True)
-        click.echo("Open Airflow in your browser. Please point your browser to http://localhost:8080 to access Airflow.")
+        docker_compose_path, project_dir = get_docker_compose_paths()
+        docker_compose_command('call', 'up', ['-d'])
```

Comment:
why -d? for now we need the user to see logs in front of them in case there are issues

Reply:
But then the **liminal stop** doesn't really take a role. It will make them to use ctrl+c.
Maybe should I add a flag for this behaviour?

Reply:
where is the stdout and err go? how can the user see them?
what feedback do users get that the process is successfully up and running?